waypoint_planner/src/velocity_set_lanelet2/velocity_set_lanelet2.cpp

```diff
@@ -681,6 +683,19 @@ int main(int argc, char** argv)
   {
     ros::spinOnce();
 
+    geometry_msgs::TransformStamped map_to_lidar_tf;
+
+    try
+    {
+        map_to_lidar_tf = tfBuffer.lookupTransform("map", "lidar", ros::Time::now(), ros::Duration(2.0));
+        vs_info.setLocalizerPose(&map_to_lidar_tf);
+    }
+    catch(tf2::TransformException &ex)
+    {
+        ROS_WARN("%s", ex.what());
```

Comment:
This message shows only about the tf error. It is better to add more information here. For example,

```
ROS_WARN("Failed to get map->lidar transform. skip computation: %s", ex.what());
```

(same for the `velocity_set.cpp`)